src/App.tsx

```diff
@@ -14,11 +15,90 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Prop = {
+  selectedGood: string[];
```

Comment:
```suggestion
  selectedGoods: string[];
```
Це ж масив, в якому буде дедкілька вибраних товарів, а не один